mlflow/spark.py

```diff
@@ -660,6 +673,34 @@ def _load_model(model_uri, dfs_tmpdir_base=None):
     return PipelineModel.load(model_uri)
 
 
+def _log_model_via_mlflowdbfs(artifact_repo, artifact_path, model):
+    mlflowdbfs_path = databricks_utils.get_mlflowdbfs_path(artifact_repo.run_id, artifact_path)
+    databricks_utils.set_databricks_host_creds_to_credential_context(
+        artifact_repo.databricks_profile_uri
+    )
+
+    try:
+        model.save(mlflowdbfs_path)
+    finally:
+        databricks_utils.clear_credential_context()
+
+
+def _load_model_via_mlflowdbfs(artifact_repo, artifact_path):
+    from pyspark.ml.pipeline import PipelineModel
+
+    mlflowdbfs_path = databricks_utils.get_mlflowdbfs_path(artifact_repo.run_id, artifact_path)
+    databricks_utils.set_databricks_host_creds_to_credential_context(
+        artifact_repo.databricks_profile_uri
+    )
+
+    try:
+        model = PipelineModel.load(mlflowdbfs_path)
+    finally:
+        databricks_utils.clear_credential_context()
+
+    return model
```

Comment:
These methods need docstrings, and we should document the assumptions that these methods make (i.e. mlflowdbfs is available *and* the artifact location associated with the Run lies within `dbfs:/databricks/mlflow-tracking`).